package.json

```diff
@@ -41,37 +41,37 @@
   "peerDependencies": {},
   "optionalDependencies": {},
   "dependencies": {
-    "class-validator": "0.13.1",
-    "express": "4.17.1",
+    "class-validator": "0.14.0",
+    "express": "4.18.2",
     "lodash": "4.17.21",
-    "nestjs-typeorm-paginate": "2.2.6",
-    "typescript": "4.4.4"
+    "nestjs-typeorm-paginate": "4.0.3",
```

Comment:
Yes, actually this was the package complaining after we bumped our direct typeorm dependency.
I did a very light research on this package regarding compatibilities between
latest NestJS and TypeORM but as you say probably worth looking more in depth and even tagging a RC so I can locally test it in LG. 
Thoughts?

Reply:
yep, I'd at least try installing this branch of `nestjs-base-service` from the GH branch and running LG tests with this setup, so that you can get an idea of outright explosions, and hopefully be able to verify that no functionality is broken. not very scientific, I know, but...